src/components/shared/svg/index.jsx

```diff
@@ -0,0 +1 @@
+export { default } from './svg';
```

Comment:
is this file doing anything? if not lets delete it. Alternatively you can export the rest of the icons from this index file so that the imports in the other components will be cleaner. For example if this file contains these exports:
export { default as Arrow} from './Arrow'
export { default as Hamburger} from './Hamburger'

Then in another file your imports will look like this:

import { Hamburger, Arrow } from 'svg'